docs/feature-gates.md

```diff
@@ -81,3 +82,15 @@ We are currently working on adding documentation for this feature.
 This feature can only be used in "encap" mode when the Geneve tunnel type is
 being used. Note that this is the default configuration for both Linux and
 Windows.
+
+### Flow Exporter
+
+`Flow Exporter` is a feature that runs as part of Antrea Agent, and enables
```

Comment:
`Flow Exporter` is a feature that runs as part of the Antrea Agent, and enables network flow visibility into a Kubernetes cluster.